proposals/CSP.md

```diff
@@ -1,94 +1,76 @@
 # WebAssembly Content Security Policy
 
-This proposal attempts to homogenize existing WebAssembly implementation's
-handling of Content Security Policy (CSP).
+This note describes a recommendation to the WebAppSec WG to extend Content Security Policy (CSP) to support compiling and executing WebAssembly modules.
```

Comment:
Note that generally the way this is done is that you would PR https://github.com/w3c/webappsec-csp/ (and file an issue to start the conversation). It's fine to have an explainer, but I don't think you can expect them to pick up the work.

Reply:
This is completely understood.